config-service-impl/src/main/java/org/hypertrace/config/service/ConfigServiceGrpcImpl.java

```diff
@@ -57,7 +58,7 @@ public void getConfig(GetConfigRequest request,
         config = merge(config, contextSpecificConfig);
       }
       responseObserver.onNext(GetConfigResponse.newBuilder()
-          .setConfig(config)
+          .setConfig(nullSafeValue(config))
```

Comment:
This is just being defensive right? Our implementation no longer returns a null?

Reply:
Yes, we're just being defensive here. The implementation no longer returns a null value.